README.md

```diff
@@ -9,19 +9,11 @@ The package can be installed using pip:
 
     pip install twoaxistracking
 
-The main non-standard dependency is `shapely`, which handles the geometric operations. It is recommended to install `shapely` using conda, which can be done by executing the following command in an Anaconda Prompt:
-
-    conda install shapely
-
-The solar modeling library `pvlib` is recommended for calculating the solar position and can be installed by the command:
-
-    pip install pvlib
-
 ## Citing
 If you use the package in published work, please cite:
-> Adam R. Jensen et al. 2022.
-> "Self-shading of two-axis tracking solar collectors: Impact of field layout, latitude, and aperture shape."
-> Accepted in Solar Energy.
+> Adam R. Jensen, Ioannis Sifnaios, Janne Dragsted, and Simon Furbo. "Self­shading of two­axis
```

Comment:
```suggestion
> Adam R. Jensen, Ioannis Sifnaios, Janne Dragsted, and Simon Furbo. "Self­-shading of two­-axis
```

Reply:
also seems like the author orders in this PR are different from the published paper?

Reply:
You're correct. I've written enough papers, that I've started switching around author order around a bit... 😄